### PR TITLE
fix: split path string manually

### PR DESCRIPTION
**Pull request by chozandrias76**
This should allow users on Linux to parse the virtual path correctly when loading a save file.

### CODE REVIEW

File: src/util/regulation.rs [hunk 1]
```diff
@@ -88,7 +88,7 @@ impl Regulation {
 
         for file in &res.files {
             let path = PathBuf::from_str(&file.name).unwrap();
```
Comment (ClayAmore) [inline src/util/regulation.rs:90]: You can drop the Pathbuf and just split the `file_name` directly. I believe it's a String or &str.

File: src/util/regulation.rs [hunk 2]
```diff
@@ -88,7 +88,7 @@ impl Regulation {
...
-            let file_name = path.file_stem().unwrap().to_str().unwrap();
+            let file_name = path.file_stem().unwrap().to_str().unwrap().split("\\").last().expect("Could not find content at end of path");
```
Comment (chozandrias76) [inline src/util/regulation.rs:91]:
```suggestion
            let file_name = &file.name.split("\\").last().expect("Could not locate file name").split(".").nth(0).expect("Could not locate file name without extension");
```

Comment (chozandrias76) [reply src/util/regulation.rs:91]: @ClayAmore What do you think of these?

Comment (chozandrias76) [reply src/util/regulation.rs:91]: It works in my VM

Comment (ClayAmore) [reply src/util/regulation.rs:91]: Gus voice: It's acceptable. 

Have to circle back to this and figure out why file_stem couldn't strip the path on linux.

**Pull request merged**